nextflow.config

```diff
@@ -1,112 +1,132 @@
-// Profiles
-profiles {
-    standard {
-        process {
-            executor = 'local'
-            withName: 'ENCYCLOPEDIA_.*' {
-                container = 'docker-encyclopedia'
-            }
-            withName: 'MSCONVERT' {
-                container = 'docker-msconvert'
-            }
-            withName: 'MSSTATS' {
-                container = 'docker-msstats'
-            }
-        }
-    }
-    cloud {
-        process {
-            executor = 'awsbatch'
-            errorStrategy = 'retry'
-            maxRetries = 3
-            withName: 'ENCYCLOPEDIA_LOCAL' {
-                container = '622568582929.dkr.ecr.us-west-2.amazonaws.com/docker-encyclopedia'
-                queue = 'terraform-nextflow-medium-size-spot-batch-job-queue'
-                memory = 31100.MB
-                cpus = 4
-            }
-            withName: 'ENCYCLOPEDIA_GLOBAL' {
-                container = '622568582929.dkr.ecr.us-west-2.amazonaws.com/docker-encyclopedia'
-                queue = 'terraform-nextflow-large-size-spot-batch-job-queue'
-                memory = 62000.MB
-                cpus = 4
-            }
-            withName: 'MSCONVERT' {
-                container = '622568582929.dkr.ecr.us-west-2.amazonaws.com/docker-msconvert'
-                queue = 'terraform-nextflow-medium-size-spot-batch-job-queue'
-                memory = 7700.MB
-                cpus = 2
-            }
-            withName: 'MSSTATS' {
-                container = '622568582929.dkr.ecr.us-west-2.amazonaws.com/docker-msstats'
-                queue = 'terraform-nextflow-medium-size-spot-batch-job-queue'
-                memory = 15400.MB
-                cpus = 2
-            }
-        }
-    }
-}
-
-// Docker Parameters
-docker {
-    enabled = true
-}
-
 // Pipeline Parameters
 params {
-    experimentName = 'purple-pig'
-    publish_dir = 'results'
-    store_dir = 'store'
+    // Input options
+    ms_file_csv = null
+    fasta = null
+    dlib = null
+
+    // Output directories
+    result_dir = 'results'
+    report_dir = 'reports'
     mzml_dir = 'mzml'
-    ms_file_csv = ''
-    use_msstats = true
+
+    // Email to alert on completion.
+    email = null
+
+    // Aggregate groups into single analysis.
     aggregate = false
-    email = ''
+    agg_name = 'aggregated'
```

Comment:
Non-blocking question: What is this purpose of this parameter?

Reply:
This one is a little weird...

Essentially the `group` column in the `ms_file_csv` file is used to specify which chromatogram library mzML files should be used to analyze which quantitative mzML files. However, sometimes we want to aggregate all of the results together in the end, say for database building or when we analyze chromatin fractions with fraction-specific chromatogram libraries. The pipeline handles this by essentially funneling all of the local EncyclopeDIA searches of the quantitative mzML files into a new group, which has the name defined by `agg_name`.

The only reason this parameter exists is for the rare case that the name `aggregated` conflicts with an existing group that someone might have.